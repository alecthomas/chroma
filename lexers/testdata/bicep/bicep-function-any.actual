publicIPAddress: any((pipId == '') ? null : {
  id: pipId
})