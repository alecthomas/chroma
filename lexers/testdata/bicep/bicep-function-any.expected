[
  {"type":"NameProperty","value":"publicIPAddress"},
  {"type":"Punctuation","value":":"},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameFunction","value":"any"},
  {"type":"Punctuation","value":"(("},
  {"type":"Text","value":"pipId"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Operator","value":"=="},
  {"type":"TextWhitespace","value":" "},
  {"type":"LiteralString","value":"''"},
  {"type":"Punctuation","value":")"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Punctuation","value":"?"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"null"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Punctuation","value":":"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Punctuation","value":"{"},
  {"type":"TextWhitespace","value":"\n  "},
  {"type":"NameProperty","value":"id"},
  {"type":"Punctuation","value":":"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"pipId"},
  {"type":"TextWhitespace","value":"\n"},
  {"type":"Punctuation","value":"})"}
]
